src/views/Maintenancepage.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <main class="maintenance-page">
+        <div class="maintenance-page_content level is-mobile">
```

Comment:
@gonzalo-plaza Why are you using level and is-mobile, if you cadd add this styles using the class _content  and is not showing correctly on my laptop, please review in others desktop views
![image](https://user-images.githubusercontent.com/47737142/106762624-4b264b00-6636-11eb-9f56-b8be6e79b65f.png)